batch/batch/utils.py

```diff
@@ -10,7 +10,11 @@
 
 
 def cost_from_msec_mcpu(msec_mcpu):
-    cost_per_core_sec = 0.013 / 3600
+    # https://cloud.google.com/compute/all-pricing
+    # n1-standard-16: $0.16 / 16 cores = $0.01 / core*hour
+    # 100 GB persistent disk: $0.17 GB/month
+    # = 0.17 * 100 * (1 / 30) * (1 / 24) * (1 / 16) = 0.0014757
```

Comment:
The mean month is 30.42 days:
```
>>> (7 * 31 + 4 * 30 + 28)/12.
30.416666666666668
```
That's a 1.4% cost center for disk (which is itself only 10% of total cost), not sure how much it matters.

I think, if we're using 5 sigfigs in our output, we should probably use at least that many in our input parameters as well.
```
In [3]: mean_month = (7 * 31 + 4 * 30 + 28)/12. 
   ...: 0.17 * 100 * (1 / mean_month) * (1 / 24.) * (1 / 16.)                                                                                                                                               
Out[3]: 0.0014554794520547942
```